// cr-pcd/esm2015/lib/cr-pcd.service.js
/**
 * @fileoverview added by tsickle
 * Generated from: lib/cr-pcd.service.ts
 * @suppress {checkTypes,constantProperty,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Injectable } from '@angular/core';
import { provinces, cantons, districts } from './data';
import * as i0 from "@angular/core";
export class CrPcdService {
    constructor() { }
    /**
     * @return {?}
     */
    getProvinces() {
        return provinces;
    }
    /**
     * @param {?} provinceId
     * @return {?}
     */
    getCantons(provinceId) {
        return cantons[provinceId];
    }
    /**
     * @param {?} cantonId
     * @return {?}
     */
    getDistricts(cantonId) {
        return districts[cantonId];
    }
}
CrPcdService.decorators = [
    { type: Injectable, args: [{
                providedIn: 'root'
            },] }
];
/** @nocollapse */
CrPcdService.ctorParameters = () => [];
/** @nocollapse */ CrPcdService.ngInjectableDef = i0.defineInjectable({ factory: function CrPcdService_Factory() { return new CrPcdService(); }, token: CrPcdService, providedIn: "root" });
