// cr-pcd/esm5/lib/cr-pcd.service.js
/**
 * @fileoverview added by tsickle
 * Generated from: lib/cr-pcd.service.ts
 * @suppress {checkTypes,constantProperty,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Injectable } from '@angular/core';
import { provinces, cantons, districts } from './data';
import * as i0 from "@angular/core";
var CrPcdService = /** @class */ (function () {
    function CrPcdService() {
    }
    /**
     * @return {?}
     */
    CrPcdService.prototype.getProvinces = /**
     * @return {?}
     */
    function () {
        return provinces;
    };
    /**
     * @param {?} provinceId
     * @return {?}
     */
    CrPcdService.prototype.getCantons = /**
     * @param {?} provinceId
     * @return {?}
     */
    function (provinceId) {
        return cantons[provinceId];
    };
    /**
     * @param {?} cantonId
     * @return {?}
     */
    CrPcdService.prototype.getDistricts = /**
     * @param {?} cantonId
     * @return {?}
     */
    function (cantonId) {
        return districts[cantonId];
    };
    CrPcdService.decorators = [
        { type: Injectable, args: [{
                    providedIn: 'root'
                },] }
    ];
    /** @nocollapse */
    CrPcdService.ctorParameters = function () { return []; };
    /** @nocollapse */ CrPcdService.ngInjectableDef = i0.defineInjectable({ factory: function CrPcdService_Factory() { return new CrPcdService(); }, token: CrPcdService, providedIn: "root" });
    return CrPcdService;
}());
export { CrPcdService };
